zomes/signal/coordinator/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+use hdk::prelude::*;
+
+#[hdk_extern]
+fn emit_10k_signals() -> ExternResult<()> {
+    for _ in 0..10_000 {
+        emit_signal("test payload")?;
+    }
```

Comment:
```suggestion
    for i in 0..10_000 {
        emit_signal(format!("test payload {i}"))?;
    }
```

Just for debugging? Feel free to ignore this : )